### PR TITLE
fix: Make user updates default to skip ETag checks (wildcard match)

### PR DESCRIPTION
**Pull request by alfrunes**
This commit restores the old behavior by default.

### CODE REVIEW

**Comment by alfrunes** [comment]
@mender-test-bot start pipeline --fast

**Review by kjaskiewiczz — APPROVED** [review]
looks good, but since I was not able to catch issues with previous fix, maybe it would be good to wait for another pair of eyes before merging?
@merlin-northern can you please take a look?

**Comment by alfrunes** [comment]
@mender-test-bot start pipeline

**Pull request merged**
